README.md

```diff
@@ -0,0 +1,59 @@
+# [Access to Memory](https://www.accesstomemory.org)
+
+Developed and maintained by [Artefactual Systems](https://www.artefactual.com/)
+
+AtoM (short for Access to Memory) is a web-based, open source application for
+standards-based archival description and access. The application is
+multilingual and multi-repository. First commissioned by the International
+Council on Archives ([ICA](https://www.ica.org)) to make it easier for
+archival institutions worldwide to put their holdings online using the ICA’s
+descriptive standards, the project has since grown into an internationally
+used community-driven project. Learn more at:
+
+* https://www.accesstomemory.org
+
+You are free to copy, modify, and distribute AtoM with attribution under the
+terms of the AGPLv3 license. See the [LICENSE](LICENSE) file for details.
+
+## Installation
+
+**Production installation**
+
+AtoM is intended to be installed using a Linux-based operating system. We use
+Ubuntu LTS releases in development and testing, but users have successfully
+installed on other distributions as well.
+
+* [Linux installation guides](https://www.accesstomemory.org/docs/latest/admin-manual/installation/linux/linux/)
+
+For other O/S installs, we recommend virtualization.
```

Comment:
Does this need a bit more explanation?

Reply:
I'm gonna say no - we explain more in the docs. I don't think the README should be too long, and it will likely mostly be developers who are looking at the code repo anyway, who will be familiar with the term